src/scripts/main.js

```diff
@@ -1,7 +1,95 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const startButton = document.querySelector('.start');
+const startMessage = document.querySelector('.message-start');
+const loseMessage = document.querySelector('.message-lose');
+const winMessage = document.querySelector('.message-win');
+const gameScore = document.querySelector('.game-score');
+
+startButton.addEventListener('click', () => {
+  if (startButton.classList.contains('start')) {
+    game.start();
+    setState();
+
+    startButton.classList = 'button restart';
+    startButton.innerHTML = 'Restart';
+    startMessage.classList.add('hidden');
+    loseMessage.classList.add('hidden');
+    winMessage.classList.add('hidden');
+
+    getScore();
+  } else if (startButton.classList.contains('restart')) {
+    game.restart();
+    setState();
+
+    startButton.classList = 'button start';
+    startButton.innerHTML = 'Start';
+    startMessage.classList.remove('hidden');
+    loseMessage.classList.add('hidden');
+    winMessage.classList.add('hidden');
+
+    getScore();
+  }
+});
+
+document.addEventListener('keyup', (e) => {
+  switch (e.key) {
+    case 'ArrowUp':
+    case 'w':
+      game.moveUp();
+      setState();
+      break;
+    case 'ArrowDown':
+    case 's':
+      game.moveDown();
+      setState();
+      break;
+    case 'ArrowLeft':
+    case 'a':
+      game.moveLeft();
+      setState();
+      break;
+    case 'ArrowRight':
+    case 'd':
+      game.moveRight();
+      setState();
+      break;
+  }
+
+  getScore();
+  updateMessages();
```

Comment:
The `updateMessages()` function call is outside of the switch statement. If none of the cases match, `updateMessages()` will still be executed, which might not be necessary. Consider placing it inside a default case or after a check that ensures a valid move was made.